### PR TITLE
This might fix what's been breaking auth tests

### PR DESCRIPTION
**Pull request by imbstack**
Gotta run now, this is messy, but is what we need to do to fix the broken auth tests.

### CODE REVIEW

File: lib/client.js [hunk 1]
```diff
@@ -322,7 +322,9 @@ exports.createClient = function(reference, name) {
             if (reportStats) {
               reportStats(true, 'http-' + res.status);
             }
-            if(res.headers['content-type'].indexOf('application/json') === -1 || !res.body){
+            if((_.includes(res.headers, 'content-type') &&
```
Comment (jonasfj) [inline lib/client.js]: `_.includes(res.headers, 'content-type')` checks if there is a value called 'content-type', not if there is a key.

Either use `_.includes(res.headers['content-type'], 'application/json')` or do `if (res.headers['content-type'] && res.headers['content-type'].indexOf(...`

That's just my guess...

Comment (jonasfj) [inline lib/client.js]: hmm... maybe I'm reading the docs wrong... Or are we missing some tests, because ideally this should be caught by travis...

**Comment by imbstack** [comment]
Oops, you're totally right. This is the correct fix.

**Comment by jonasfj** [comment]
R+

**Comment by djmitche** [comment]
R+!  Please land, release a new version, and update tc-tools to use that new version as a minimum.

**Pull request merged**